en/power_module/README.md

```diff
@@ -19,6 +16,7 @@ This section provides links/information about supported power modules and power
   * [Holybro PM02](../power_module/holybro_pm02.md)
   * [Holybro PM07](../power_module/holybro_pm07_pixhawk4_power_module.md)
   * [Holybro PM06 V2](../power_module/holybro_pm06_pixhawk4mini_power_module.md)
+  * [Sky-Drones SmartAP PDB](../power_module/sky-drones_smartap-pdb.md)
```

Comment:
This list is a mix of pure analog voltage and current power modules and power modules/PDBs.
Do you think it worth splitting out into PDBs vs pure power modules. I am "somewhat inclined,. since power modules ado simplify things a lot!

Reply:
I think it's a good idea to split for Power Modules and Power Distribution Boards, especially since there are a few from Holybro already existing there. The final decision is up to you but I'm definitely up for this proposal.